crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -564,53 +600,52 @@ impl<'w, T: Fetch<'w>> Fetch<'w> for OptionFetch<T> {
     }
 }
 
-/// Flags on component `T` that happened since the start of the frame.
+/// Flags on component `T` that happened since the last execution of this system.
 #[derive(Clone)]
-pub struct Flags<T: Component> {
-    flags: ComponentFlags,
+pub struct Counters<T: Component> {
```

Comment:
I don't think `Flags` is meaningful here, since the implementation has nothing to do with flags now. `Trackers` is fine, but could be confused with other trackers, for instance for removed entities. How about `ChangeTrackers`?

Reply:
`ChangeTrackers` is much more descriptive and is my favorite.

Reply:
Yup I dig `ChangeTrackers`

Reply:
Renamed in def2980e75db4343aca2db5872585949d18c9b5e.